library/Odesk/Phystrix/RequestLog.php

```diff
@@ -49,4 +49,72 @@ public function addExecutedCommand(AbstractCommand $command)
     {
         $this->executedCommands[] = $command;
     }
+
+    /**
+     * Formats the log of executed commands into a string usable for logging purposes.
+     *
+     * @return string
+     */
+    public function getExecutedCommandsAsString()
+    {
+        try {
+            $output = "";
+            $executedCommands = $this->getExecutedCommands();
+            $aggregatedCommandsExecuted = array();
+            $aggregatedCommandExecutionTime = array();
+
+            /** @var \Odesk\Phystrix\AbstractCommand $executedCommand */
+            foreach ($executedCommands as $executedCommand) {
+                $display = $executedCommand->getCommandKey() . "[";
+                $events = $executedCommand->getExecutionEvents();
+
+                if (count($events) > 0) {
+                    foreach ($events as $event) {
+                        $display .= "{$event}, ";
+                    }
+                    $display = substr($display, 0, -2);
+                } else {
+                    $display .= "Executed";
+                }
+
+                $display .= "]";
+
+                if (!isset($aggregatedCommandsExecuted[$display])) {
+                    $aggregatedCommandsExecuted[$display] = 0;
+                }
+
+                $aggregatedCommandsExecuted[$display] = $aggregatedCommandsExecuted[$display] + 1;
+
+                $executionTime = $executedCommand->getExecutionTimeInMilliseconds();
+
+                if ($executionTime < 0) {
+                    $executionTime = 0;
+                }
+
+                if (isset($aggregatedCommandExecutionTime[$display]) && $executionTime > 0) {
+                    $aggregatedCommandExecutionTime[$display] = $aggregatedCommandExecutionTime[$display] + $executionTime;
+                } else {
+                    $aggregatedCommandExecutionTime[$display] = $executionTime;
+                }
+            }
+
+            foreach ($aggregatedCommandsExecuted as $display => $count) {
+                if (strlen($output) > 0) {
+                    $output .= ", ";
+                }
+
+                $output .= "{$display}";
+
+                $output .= "[" . $aggregatedCommandExecutionTime[$display] . "ms]";
+
+                if ($count > 1) {
+                    $output .= "x{$count}";
+                }
+            }
+
+            return $output;
+        } catch (\Exception $e) {
```

Comment:
in original Java implementation this exception is logged, here we just hide it. @mrook would it make sense to simply remove the try-catch block so it gets propagated up?